src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+@import "../utils/variables";
+
+.calendar {
+  /* Display Flex & Box Model */
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: #{$day-gap};
+  width: #{$calendar-width};
+  height: auto;
+  padding: #{$calendar-padding};
+  padding-top: #{$calendar-padding-top};
+
+  /* for testing bg */
+  // background-color: beige;
+
+  &__day {
```

Comment:
I think it would be useful to have function for setting day square size - maybe something like @function square()

Reply:
I'm kinda confused, bc I don't understand how it should work? 
I used:
    width: #{$day-size};
    height: #{$day-size};
    
And also in variables:
   $day-size: <number>;